documentation/release-instructions.md

```diff
@@ -0,0 +1,30 @@
+Releases are available on the [downloads page](https://github.com/AnalyticalGraphicsInc/ion-blender-exporter/releases). There is no release manager; instead, our community shares the responsibility.  Any committer can create the release for a given month, and at any point, they can pass the responsibility to someone else, or someone else can ask for it.  This spreads knowledge, avoids stratification, avoids a single point of failure, and is beautifully unstructured.
+## Packaging, Testing, Release
+### Package
+1. Send a courtesy message in the [slack](https://cesium.slack.com) to let people know you're about to start the release process.
```

Comment:
Can probably remove all items about slack, since it's not a public channel.

Reply:
@mramato Slack is in the cesium release notes?

Reply:
> Works great, a couple of comments.
> 
> What are the licenses for the test models you added and where did they come from? This information should be included in a ThirdParty.md. We should also have an entry in there for whatever top-level package we are including in the third_party folder.

They have always been there. I just moved them.

Reply:
Okay, sorry I missed it the first time.  We still need to properly document them

Reply:
I was thinking and I will remove them from the repo but have it be a part of the developer setup process.

Reply:
Maybe I'm missing something, but I don't see any mention here: https://github.com/AnalyticalGraphicsInc/cesium/wiki/Release-Guide I don't think we reference it in any open source project (but I could be wrong)

Reply:
My bad mistook forum as slack `Send a courtesy message to the forum to let people know you're about to start the release process.`

Reply:
Yeah, these smaller projects can have a little lighter of a release process. No big deal.